collection-spec/collection-spec.md

```diff
@@ -32,11 +32,14 @@ Implementations are encouraged, however, as good effort will be made to not chan
 | license      | string            | **REQUIRED.** Collection's license(s) as a SPDX [License identifier](https://spdx.org/licenses/) or [expression](https://spdx.org/spdx-specification-21-web-version#h.jxpfx0ykyb60) or `proprietary` if the license is not on the SPDX license list. Proprietary licensed data SHOULD add a link to the license text, see the `license` relation type. |
 | providers    | [Provider Object] | A list of providers, which may include all organizations capturing or processing the data or the hosting provider. Providers should be listed in chronological order with the most recent provider being the last element of the list. |
 | extent       | Extent Object     | **REQUIRED.** Spatial and temporal extents.                  |
-| properties   | object            | Common fields across referenced items. May also be used to describe standalone collections better that don't reference any items. See the section 'Common Fields' for more information. |
+| properties   | object            | Common fields across referenced items. |
+| summaries    | Map<string, [*]|Range Object> | A map of property summaries, either a set of values or a range. |
```

Comment:
This should be <Range Object>

Reply:
From your comment not sure what exactly you want to have changed, but I assume escaping the pipe? Done!